rust/operator-binary/src/backend/mod.rs

```diff
@@ -179,7 +179,8 @@ impl SecretVolumeSelector {
         scope: &scope::SecretScope,
     ) -> Result<Vec<Address>, ScopeAddressesError> {
         use scope_addresses_error::*;
-        let cluster_domain = &pod_info.kubernetes_cluster_domain;
+        // Turn FQDNs into bare domain names by removing the trailing dot
```

Comment:
Just FYI that this will trim *all* trailing dots, not just one. Should make no difference in practice though
```suggestion
        // Turn FQDNs into bare domain names by removing the trailing dots
```